cds_ils/vocabularies/data/acq_payment_modes.json

```diff
@@ -8,5 +8,10 @@
     "type": "acq_payment_mode",
     "key": "INTERNAL_PURCHASE_REQUISITION",
     "text": "Internal Purchase Requisition"
+  },
+  {
+    "type": "acq_payment_mode",
+    "key": "MIGRATED_UNKNOWN",
```

Comment:
Can you please add a card in the board to update vocabs during the deployment process?